### DISCUSSION TITLE
Issue with NFC Integration

### DISCUSSION THREAD

**Discussion by PiyushSaklani** [alexandercerutti/passkit-generator#217] [Q&A] [answered]
Hello everyone,

I’m working on integrating the NFC feature into my Pass. The .pkpass file is generating, but it’s not opening. When I remove the NFC code, it works fine. Has anyone encountered this issue before or know how to resolve it?

Thanks in advance!

**Comment by alexandercerutti** [comment, marked as answer]
Hey @PiyushSaklani, thanks for using passkit-generator!

Did you try to follow the [Self-help guide](https://github.com/alexandercerutti/passkit-generator/wiki/Troubleshooting-(Self-help)#through-console-mac-only) I set up in the wiki?

**Reply by PiyushSaklani** [reply]
I just tested it and realized that an Enhanced Passbook certificate is required for NFC functionality. Thanks for the help!
